packages/typescript/data/snippets/typescript.json

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
do we need to declare the source of these snippets?

Reply:
I don't know how

Reply:
Maybe something like:
```ts
{
    "description": "blablabla",
    "source": "vscode ..."
    "snippets": {
        "Constructor": {
            "prefix": "ctor",
            "body": [
                "/**",
                " *",
                " */",
                "constructor() {",
                "\tsuper();",
                "\t$0",
                "}"
            ],
            "description": "Constructor"
        },
        // ...
    }
}
```
And when you `require` the file:
```ts
    function registerSnippets() {
        const snippetDefinition = require('../../data/snippets/typescript.json');
        const snippets = snippetDefinition[snippets];
        monaco.languages.registerCompletionItemProvider(TYPESCRIPT_LANGUAGE_ID, new TextmateSnippetCompletionProvider(snippets, 'ts'));
        monaco.languages.registerCompletionItemProvider(TYPESCRIPT_REACT_LANGUAGE_ID, new TextmateSnippetCompletionProvider(snippets, 'ts'));
    }
```
wdyt?